packages/site-kit/src/lib/styles/text.css

```
@@ -286,10 +286,12 @@
 }
 
 .text blockquote {
-	color: var(--sk-text-translucent);
-	background-color: rgba(255, 62, 0, 0.2);
-	border-left: 4px solid #ff3e00;
+	background: var(--sk-back-1);
+	color: var(--sk-text-1);
+	border: 1px solid var(--sk-theme-1);
+	border-radius: var(--sk-border-radius);
 	padding: 1rem;
+	filter: drop-shadow(0 0 6px hsla(15, 100%, 50%, 0.2));
```

I'm good with this change! It uses our branding color, so that's a plus, and it's not as in-your-face as before, which is a  big plus. The one suggestion I have is to offset the drop-shadow a little, it looks more pleasing to the eye like that to me and has a little more of that "elevated" style.
```suggestion
	filter: drop-shadow(2px 2px 6px hsla(15, 100%, 50%, 0.2));
```